.eslintrc.json

```diff
@@ -1,6 +1,231 @@
 {
   "extends": "./node_modules/aurelia-tools/.eslintrc.json",
   "rules": {
-    "no-new-func": 0
+    "accessor-pairs": "error",
+    "array-bracket-spacing": [
```

Comment:
Perhaps also use tools here? There are some nice strict rules here we could apply across the board. :)